tests/utilities/test_cli.py

```diff
@@ -1547,3 +1549,62 @@ def test_cli_logger_shorthand():
     with mock.patch("sys.argv", ["any.py", "--trainer.logger=False"]):
         cli = LightningCLI(TestModel, run=False)
     assert cli.trainer.logger is None
+
+
+def test_cli_auto_seeding():
+    with mock.patch("sys.argv", ["any.py"]), pytest.raises(ParserError, match="seed_everything.*does not validate"):
+        LightningCLI(
+            TestModel,
+            run=False,
+            seed_everything_default=int(np.iinfo(np.uint32).max + 1),
+            trainer_defaults={"logger": False},
+            parser_kwargs={"error_handler": None},
+        )
+    with mock.patch("sys.argv", ["any.py"]), pytest.raises(ParserError, match="seed_everything.*does not validate"):
+        LightningCLI(
+            TestModel,
+            run=False,
+            seed_everything_default=int(np.iinfo(np.uint32).min - 1),
+            trainer_defaults={"logger": False},
+            parser_kwargs={"error_handler": None},
+        )
+
+    with mock.patch("sys.argv", ["any.py"]):
+        cli = LightningCLI(TestModel, run=False, seed_everything_default=False, trainer_defaults={"logger": False})
```

Comment:
question: why do you set `logger: False` in all these cases?

Reply:
Mostly, because I copied the test from above (test_cli_logger_shorthand) and as the tests do not require a logger I just left it there. I also wanted to make sure no log_dirs or anything are created (didn't know out of my head, when exactly the loggers create their log_dir. If upon initialization or somewhere later).
I can remove it if you want.

Reply:
Yes. You can remove them

The test above set it because it was explicitly testing the logger argument.

This is not going to generate any artifacts since `run=False` is set